can-dom-events-test.js

```diff
@@ -10,15 +10,22 @@ unit.test('domEvents.addEventListener works', function (assert) {
 
 	var input = document.createElement('input');
 	var eventType = 'click';
+	var event;
+
 	var handler = function () {
 		assert.ok(true, 'event handler should be called');
 	};
 
 	domEvents.addEventListener(input, eventType, handler);
-
-	var event = new Event(eventType);
+	if (typeof Event === "function") {
+		event = new Event(eventType);
+	} else {
+		var qf = document.querySelector('#qunit-fixture');
+		qf.appendChild(input);
```

Comment:
Any reason we can't just do this all the time? It seems confusing to do this as part of the feature detection.

Reply:
No, but those [features](https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent) are [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Document/createEvent) if they will removed in future browsers versions the tests will break.

Reply:
I just mean the `appendChild`.